spec/unit/commands/activedocs_command/apply_command_spec.rb

```diff
@@ -87,16 +88,15 @@
           expect(activedocs_class).to receive(:find).with(remote: remote, ref: activedocs_ref).and_return(activedocs)
         end
 
-        context 'with no options' do
-          let(:options) { {} }
-
+        context 'with default options' do
           it 'activedocs not updated' do
+            expect(activedocs).not_to receive(:update)
```

Comment:
Not required.

If `expect` not set and activedocs receives `:update`, test will fail.

Reply:
:+1: